exe/task_wrapper.cc

```diff
@@ -44,7 +45,9 @@ int main(int argc, char *argv[])
         // Avoid atomic update to allow testing against /dev/stderr. There should never be
         // multiple processes trying to write this output.
         boost::nowide::ofstream ofs { params.get<std::string>("stderr"), std::ios::binary };
+        #ifndef _WIN32
         fs::permissions(params.get<std::string>("stderr"), FILE_PERMS);
```

Comment:
You can skip making changes to this file, shouldn't affect testing and it's more related to the spool-dir permissions (which I'll fix up separately).